docs/source/transforms.rst

```diff
@@ -356,6 +356,18 @@ Post-processing
     :members:
     :special-members: __call__
 
+`LabelFilter`
+"""""""""""""""""""""""""""""""
```

Comment:
Could you please help change the length of  """""""" to be same as the words? Same as other titles.